pages/[slug]/dashboard.tsx

```diff
@@ -125,77 +119,132 @@ const MemberList = ({
 const IDPList = ({
   user,
   saml_connections,
-}: Pick<Props, "user" | "saml_connections">) => {
-  const [idpName, setIDPName] = useState("");
-  const router = useRouter();
+  oidc_connections }: Pick<Props, "user" | "saml_connections" | "oidc_connections">) => {
+    const [idpNameSAML, setIdpNameSAML] = useState("");
+    const [idpNameOIDC, setIdpNameOIDC] = useState("");
+    const [isSaml, setIsSaml] = useState(true);
```

Comment:
instead of `isSaml` I wonder if we can have some enum that tells us which SSO method we are using

Reply:
✅